keras_contrib/backend/theano_backend.py

```diff
@@ -117,6 +121,24 @@ def extract_image_patches(X, ksizes, strides, border_mode="valid", dim_ordering=
     return patches
 
 
+def depth_to_space(input, scale):
```

Comment:
depth_to_space might need its own unit test? Although is called in the pixel upscaling test.